src/js/components/Form/FormContext.js

```diff
@@ -14,6 +14,11 @@ const useFormInput = (_, valueProp, initialValue) => {
     nextValue => {
       if (initialValue !== undefined) setValue(nextValue);
     },
+    () => {
+      const nextValue = { ...value };
```

Comment:
I don't think `valueProp`, and thus `value`, is necessarily an object.

I think this function can be a no-op in this situation`() => {}`.